packages/mobile/package.json

```diff
@@ -19,6 +19,9 @@
     "@react-navigation/drawer": "^5.3.4",
     "@react-navigation/native": "^5.1.3",
     "@react-navigation/stack": "^5.2.6",
+    "@types/styled-components": "^5.0.1",
+    "@types/yup": "^0.26.34",
+    "@workspace-library/core": "1.0.0",
```

Comment:
We're not using core as a package anymore, please remove this.

Reply:
ready